gapis/api/vulkan/externs.go

```diff
@@ -1115,3 +1115,11 @@ func bindSparse(ctx context.Context, s *api.GlobalState, binds *QueuedSparseBind
 		}
 	}
 }
+
+func (e externs) notNullOrZero(v interface{}) bool {
```

Comment:
Please don't add externs unless they are absolutely necessary.
You should be able to just compare against `null`.